Learning how to use React Native part 2

neilSoriano opened the pull request:
Had to redo the UI because the other one was not running properly, but currently trying to get the logo to display in the right place and adding buttons to the UI

neilSoriano committed:
Learning how to use React Native 

Couldn't add the node_modules folder because Github said there's too many files.

neilSoriano committed:
Added the updated UI files

Unable to add node_modules folder because Github says there's too many files.

neilSoriano committed:
These files weren't updated in the last commitment

Triscuit07 commented:
> Had to redo the UI because the other one was not running properly, but currently trying to get the logo to display in the right place and adding buttons to the UI

Logo fixed by fixing logo space parameters from height: 150 and width: 150 --> height: 150 width: 300

neilSoriano committed:
Was able to fix the logo sizing

ryko9 committed:
Added home-menu buttons

Added buttons, working on connecting calendar.js to the "Event Calendar" button

jasonblock1 committed:
Update App.js

jasonblock1 committed:
Update App.js

ryko9 committed:
Update App.js

Added logo to login page

neilSoriano committed:
Edited monthFormat parameter

neilSoriano committed:
Removed week numbers

neilSoriano committed:
Added style to the buttons and day names to Calendar

neilSoriano committed:
Made buttons round and added Track Runners button

still working on functionality of track runners button

neilSoriano committed:
Got map to display for track runners button

ryko9 committed:
Update App.js

Added dummy Stats and Profile screen, will get UI template added for the pages

neilSoriano committed:
Added a marker/pin to the map

jasonblock1 committed:
Check In button and style changes
